src/common/components/Buttons/Button.tsx

```diff
@@ -5,6 +5,7 @@ export type TButton = {
   children?: ReactNode | string
   variant?: 'default' | 'secondary' | 'transparent' | 'navigation'
   disabled?: boolean
+  disabledBule?: boolean
```

Comment:
I'm afraid that no one will guess what is "disabledBule", and it's important to have understandable props in shared components.

Please remove this prop and pass "bg-opacity-50" and other classes you need via `className` prop

Reply:
fixed